language/Russian/strings.po

```diff
@@ -5968,7 +5987,7 @@ msgstr "Все"
 #: plugins/cheatfind/init.lua:907
 #, lua-format
 msgid "Test Cheat %08X_%02X"
-msgstr ""
+msgstr "Проветирь чит %08X_%02X"
```

Comment:
This is the format used for the name of the cheat that was newly created for testing.  Is this translation appropriate?